catroid/src/main/java/org/catrobat/catroid/content/bricks/BroadcastMessageBrick.java

```diff
@@ -95,6 +95,7 @@ public void onNewOptionSelected() {
 					@Override
 					public void onPositiveButtonClick(DialogInterface dialog, String textInput) {
 						addItem(textInput);
+						notifyDataSetChanged(activity);
```

Comment:
if you already have the activity here and if it has to be the `SpriteActivity`, which it has to be for the `ScriptFragment` to be present in it you might as well cast the activity to the `SpriteActivity` and then call the method you added to it below the `notifyDataSetChanged()` you added. would make sense to reuse it wouldn't it?